src/AstronautsTable.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import fetchJson from "./fetchJson";
+
+function AstronautsTable() {
+  const [error, setError] = useState(null);
+  const [astronauts, setAstronauts] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
+      fetchJson()
+      .then(
+        (data) => {
+          setAstronauts(data);
+          setIsLoaded(true);
+        },
+        (error) => {
+        setError(error);
+        setIsLoaded(true)
+        }
+      );
+  }, []);
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
+} else if (!isLoaded) {
+    return <div>Loading, please wait...</div>;
+} else {
+    console.log(astronauts.people);
```

Comment:
Stray print statement, and also the indentation on the `else` blocks are borked.

Reply:
I ran an auto-formatting thing to fix it!